app/helpers/i18n_helpers.rb

```diff
@@ -48,7 +65,33 @@ def t_model(model = nil)
     else
       model_name = model.class.name.underscore
     end
-    I18n::translate(model_name, :scope => [:activerecord, :models])
+    I18n::translate(model_name, :scope => [:activerecord, :models], count: count)
+  end
+
+  # Returns translated name for the given +model+ pluralized.
+  #
+  # This is the pluralization version of t_model.
+  #
+  # There needs to be one and other keys in:
+  # de.activerecord.models.account.one = Konto
+  # de.activerecord.models.account.other = Konten
+  #
+  # Example:
+  #   t_models(Account)     => 'Konten'
+  #   t_models              => 'Konten' # when called in patients_controller views
+  #
+  def t_models(model = nil)
```

Comment:
Do we really need another method if we introduce the `count` parameter in the original method?

Well, it's probably nicer to use in classic rubies as we cannot pass named parameters, so we would need something like `t_model(nil, 2)` if we want the auto-magic model lookup but plurals.

I would not opposed to require more modern Rubies and use named parameters though;-) I think this would keep the interface small.